mautic/mautic
# New look and feel

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [X]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #13137, Fixes #13141, Fixes #13155, Fixes #13154, Fixes #13153, Fixes #13152, Fixes #13157, Fixes #13138, Fixes #13139, Fixes #13140, Fixes #13142 

#### Description:

Well, the so waited redesign is here.

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Explore the header, notification panel, marketplace page, marketplace item details page, and user account

There are items that have not been touched, so it will remain on the old UI.
